### DISCUSSION TITLE
Is it possible to specify QR code version

### DISCUSSION THREAD

**Discussion by zhangyuyangseiko** [Shane32/QRCoder#418] [Q&A]
Hello @codebude and everyone,

Thank you for this awesome library and please pardon my layman knowledge on QR code. 

As stated in the title, is it possible to pass parameter to qrGenerator() and specify the QR code version? Or is appropriate version automatically picked based on character size of data string/payload?

Thank you in advance for taking the time to reply.

**Comment by julealgon** [comment]
This is something I'd also like to know that is completely absent from the documentation. @codebude can you clarify?
